### ISSUE TITLE
Constructive User Feedback - v16 UI/UX disappointing

### ISSUE THREAD

**Issue by cub-nick** [odoo/odoo#114070]
I don't know where else to put this. A comment/wishlist forum would be a great add... I think it's important to hear user feedback :)

The new UI/UX of v16 is a bit disappointing. So many UI elements that normally signify fields, sections, change of content, data to edit/enter, etc., are gone. 

For instance, the UI of sending emails.  It's just white. No signifiers of of where to click, contextually different information etc. It's confusing to look at.

This is the case with many UI elements, especially form views. Again, no hints of editable data. Keep in mind, many older users use Odoo!

There are many great new functional features that we're very happy to have! But, it seems Odoo was trying to get too modern with UI design. Efficiency, accuracy and ease of use is paramount.  Function over design in ERP.

Just some constructive feedback for future editions...  Thanks.  Still, many great new functions, though!

**Comment by peterromao** [comment]
Hello cub-nick,
There's a good fix out there that may help you. Try [https://apps.odoo.com/apps/modules/16.0/web_theme_classic/]

**Comment by Primary-Pineapple** [comment]
@cub-nick - I'm glad its not just us, it's driving all of our users nuts! They broke numerous keyboard shortcuts, and removed visual indicators of what can be clicked. I have raised several 'issue' tickets with Enterprise support. None have been addressed yet, which is not un-common for odoo support.

**Comment by cub-nick** [comment]
> Hello cub-nick,
> There's a good fix out there that may help you. Try https://apps.odoo.com/apps/modules/16.0/web_theme_classic/

Thank you @peterromao !  Will give this a try. Glad it's just not me and @Primary-Pineapple ;)

**Comment by cub-nick** [comment]
@peterromao so far so good!  Module working great.  Wasn't a fan of the purple, but easily changed in the CSS.  

@Primary-Pineapple highly recommend Peter's suggestion.  Works for me.  Easy to edit the colors if you need.

**Comment by Primary-Pineapple** [comment]
@cub-nick - Are you on enterprise? We have spent the better part of two years eliminating custom modules to make the upgrade cycle faster for us. I hate the thought of going down this path again... but until they address the issues I feel I have no choice. I am getting complaints by the bucket load today about changes they've made that are causing confusion or just don't make any sense at all. Ahhhhh!

**Comment by cub-nick** [comment]
> @cub-nick - Are you on enterprise? We have spent the better part of two years eliminating custom modules to make the upgrade cycle faster for us. I hate the thought of going down this path again... but until they address the issues I feel I have no choice. I am getting complaints by the bucket load today about changes they've made that are causing confusion or just don't make any sense at all. Ahhhhh!

I do have Odoo.SH.  We needed to extend it a bit more. But I'm using pretty light modules, nothing too fancy.  So far upgrading hasn't been an issue.  I've had more issues with customizations we've done via Studio.

**Comment by bluesprocket** [comment]
I agree that the changes to the user interface are confusing and difficult at times.  The worst part is training someone new as to where things are since there aren't really visual indicators anymore.  I'm all for advancing and modernizing the interface, admittedly so many people are used to super slick modern UIs.  But it feels like they either need to have the option to choose between the old/new interface, or provide some greater level of UI customization.

**Comment by Primary-Pineapple** [comment]
Another bug fix release this morning rolled out on odoo.sh, another issue caused by carelessness. This morning they’ve broken the scroll bar in the matching window within Bank Reconciliations. Odoo needs to slow down their release cycle and fix some of this shit. SAP/MS Dynamics be calling our name.

I swear if one more person tells me to buy a success pack…. *cries*

**Comment by cub-nick** [comment]
@Primary-Pineapple This might be a newb question, but how do you know which version of Odoo.sh you're on and when they update it?  I haven't noticed anything. I thought SH wasn't privy to the 16.x upgrades. That was only for Online and SH only gets the official main release...

But I hear you about the Success Pack. Big Same. Their go-to answer, rather than actually addressing bugs or standing behind their product.

**Comment by Primary-Pineapple** [comment]
![46B6B790-F81E-4A23-A9A2-2F17FB33FA0B](https://user-images.githubusercontent.com/36147901/228090925-a00c427d-5532-40dc-b345-7fabfd92350d.jpeg)

Odoo.sh, project settings. - The version number is the date.

**Comment by Primary-Pineapple** [comment]
They broke several things in 20230327 this morning.
